<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <link
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
      rel="stylesheet"
      integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x"
      crossorigin="anonymous"
    />

    <link href='https://api.mapbox.com/mapbox-gl-js/v2.9.1/mapbox-gl.css' rel='stylesheet' />

    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css"
    />
    
    <link rel="stylesheet" href="publication.css" />

    
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" 
    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" 
    crossorigin="anonymous">
  </script>
    <title>Anthony Snead</title>
  </head>

 

  <body>

    <!-- Navbar -->
    <nav class="navbar navbar-expand-lg bg-dark navbar-dark py-3 fixed-top">
      <div class="container">
        <a href="index.html" class="navbar-brand">Anthony Snead</a>

        <button
          class="navbar-toggler"
          type="button"
          data-bs-toggle="collapse"
          data-bs-target="#navmenu"
        >
          <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="navmenu">
          <ul class="navbar-nav ms-auto">
            <li class="nav-item">
              <a href="about.html" class="nav-link">About</a>
            </li>
            <li class="nav-item">
              <a href="research.html" class="nav-link">Research</a>
            </li>
            <li class="nav-item">
              <a href="publications.html" class="nav-link">Publications</a>
            </li>
            <li class="nav-item">
              <a href="outreach.html" class="nav-link">Outreach</a>
            </li>
          </ul>
        </div>
      </div>
    </nav>

     <!-- Publications -->
     <div class="container pt-5">
     <div class="container border border-secondary rounded text-dark p-2">
      <section id="preprints">
        <div class="container">
          <h2 class="text-center mb-4"><u>Preprints</u></h2>
          <div class="accordion accordion-flush" id="preprints">
            <!-- Item 1 -->
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pre-1"
                >
                <p style="text-indent: -36px; padding-left: 36px;">1. Cunha Neto IL, Snead AA, Landis JB, Specht CD, Onyenedum JG. (2024)
                  Ectopic cambia in Japanese wisteria (<span class="fst-italic">Wisteria floribunda</span>) vines are associated with the expression of conserved KNOX genes <span class="fst-italic">bioRxiv</span> 
                     (<a href="https://www.biorxiv.org/content/10.1101/2024.08.07.606835v2.abstract" class="link-primary" target="_blank">doi: 10.1101/2024.08.07.606835</a>)
                     <span class="fst-italic"> Under Review: Nature Communications</span> </p>
                </button>
              </h2>
              <div
                id="pre-1"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Abstract:</span> Secondary growth is a conserved mechanism that gives rise to 
                      vascular tissues produced via a single vascular cambium. Molecular mechanisms underlying this process are characterized 
                      mostly in model species bearing typical vascular architecture, while the genetics underlying ecologically-important 
                      atypical vascular architectures remain unexplored. We use developmental anatomy, comparative transcriptomics, and 
                      molecular evolutionary analyses to address this knowledge gap, investigating how multiple ectopic cambia (EC) form
                       in the woody vine Japanese wisteria. Anatomical studies show EC in Japanese wisteria arise from cortical parenchyma,
                        while cambium-specific transcriptome comparisons reveal that genes acting as regulators of typical cambium development
                         in model species are likewise associated with atypical EC development. Gene trees of KNOX proteins indicate duplication
                          events may contribute to EC formation, including a Fabaceae-specific duplication of KNAT6 detected as under positive
                           selection. These findings reveal insights into the genetics of EC formation, advancing our understanding of the 
                           development of complex vascular traits.
                    </p>
                </div>
              </div>
            </div>
            </div>
            </div>
            </section>
            <!-- <section id="Accepted">
              <div class="container">
                <h2 class="text-center mb-4"><u>Accepted</u></h2>
                <div class="accordion accordion-flush" id="publications"> -->
                  <!-- Item 1 -->
                  <!--<div class="accordion-item">
                      <h2 class="accordion-header">
                        <button
                          class="accordion-button collapsed"
                          type="button"
                          data-bs-toggle="collapse"
                          data-bs-target="#acc-1"
                        >
                        <p style="text-indent: -36px; padding-left: 36px;">1. <span style="font-weight:bold">Snead AA</span>, Tatarenkov A, Taylor DS, Marson K Earley RL. 
                          (202X) Centrality to the Metapopulation is more Important for Population Genetic Diversity than Habitat Area or Fragmentation. <span class="fst-italic">Accepted: Biology Letters</span></p>
                        </button>
                      </h2>
                      <div
                        id="acc-1"
                        class="accordion-collapse collapse"
                        data-bs-parent="#publications"
                      >
                        <div class="accordion-body">
                            <p><span style="font-weight:bold">Abstract:</span> Drift and gene flow affect genetic diversity. Given that the 
                              strength of genetic drift increases as population size decreases, management activities have focused on 
                              increasing population size through preserving habitats to preserve genetic diversity. Few studies have 
                              empirically evaluated the impacts of drift and gene flow on genetic diversity. <span class="fst-italic">Kryptolebias marmoratus</span>, 
                              henceforth ‘rivulus’, is a small killifish restricted to fragmented New World mangrove forests with gene 
                              flow primarily associated with ocean currents. Rivulus form distinct populations across patches, making 
                              them a well-suited system to test the extent to which habitat area, fragmentation, and connectivity are 
                              associated with genetic diversity. Using over 1,000 individuals genotyped at 32 microsatellite loci, high 
                              resolution landcover data, and oceanographic simulations with graph theory, we demonstrate that centrality 
                              (connectivity) to the metapopulation is more strongly associated with genetic diversity than habitat area or 
                              fragmentation. By comparing models with and without centrality standardized by the source population’s genetic 
                              diversity, our results of habitat configuration. suggest that metapopulation centrality is critical to genetic 
                              diversity regardless of the diversity of adjacent populations. While we find evidence that habitat area and 
                              fragmentation are related to genetic diversity, centrality is always a significant predictor with a larger 
                              effect than any measure </p>
                        </div>
                      </div>
                    </div>
                </div>
              </div>
            </section>-->
            
      <section id="publications">
        <div class="container">
          <h2 class="text-center mb-4"><u>Publications</u></h2>
          <div class="accordion accordion-flush" id="publications">
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pub-10"
                >
                <p style="text-indent: -36px; padding-left: 36px;">10. <span style="font-weight:bold">Snead AA</span>, Meng F, Largotta N, Winchell KM, Levine BA. 
                  <span class="fst-italic">Scientific Data</span>
                     (<a href="https://www.nature.com/articles/s41597-025-04854-8" class="link-primary" target="_blank">doi: 10.1038/s41597-025-04854-8</a>)</p>
                </button>
              </h2>
              <div
                id="pub-10"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Abstract:</span> The spotted lanternfly (<i>Lycorma delicatula</i>) is a planthopper species 
                      (Hemiptera: Fulgoridae) native to China but invasive in South Korea, Japan, and the United States where it is a significant threat
                       to agriculture. Genomic resources are critical to both management of this species and understanding the genomic characteristics of
                        successful invaders. We report an annotated, haplotype-phased, chromosome-level genome assembly for the spotted lanternfly using
                         PacBio long-read sequencing, Hi-C technology, and RNA-seq. The 2.2 Gbp genome comprises 13 chromosomes, and whole genome resequencing
                          of eighty-two adults indicated chromosome four as the sex chromosome and a corresponding XO sex-determination system. We 
                          identified over 12,000 protein-coding genes and performed functional annotation, facilitating the identification of candidate 
                          genes that may hold importance for spotted lanternfly control. The assemblies and annotations were highly complete with over 
                          96% of BUSCO genes complete regardless of the database (i.e., Eukaryota, Arthropoda, Insecta). This reference-quality genome 
                          will serve as an important resource for development and optimization of management practices for the spotted lanternfly and 
                          invasive species genomics as a whole.</p>
                </div>
              </div>
            </div>
            <!-- Item 1 -->
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pub-9"
                >
                <p style="text-indent: -36px; padding-left: 36px;">9. Kosch et al. [including <span style="font-weight:bold">Snead AA</span>]. 
                  The Amphibian Genomics Consortium: advancing genomic and genetic resources for amphibian research and conservation. <span class="fst-italic">BMC Genomics</span>
                     (<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-10899-7" class="link-primary" target="_blank">doi: 10.1186/s12864-024-10899-7</a>)</p>
                </button>
              </h2>
              <div
                id="pub-9"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Abstract:</span> Amphibians represent a diverse group of tetrapods, marked by deep divergence times 
                      between their three systematic orders and families. Studying amphibian biology through the genomics lens increases our understanding
                       of the features of this animal class and that of other terrestrial vertebrates. The need for amphibian genomic resources is more urgent
                        than ever due to the increasing threats to this group. Amphibians are one of the most imperiled taxonomic groups, with approximately
                         41% of species threatened with extinction due to habitat loss, changes in land use patterns, disease, climate change, and their 
                         synergistic effects. Amphibian genomic resources have provided a better understanding of ontogenetic diversity, tissue regeneration,
                          diverse life history and reproductive modes, anti-predator strategies, and resilience and adaptive responses. They also serve as 
                          essential models for studying broad genomic traits, such as evolutionary genome expansions and contractions, as they exhibit the 
                          widest range of genome sizes among all animal taxa and possess multiple mechanisms of genetic sex determination. Despite these 
                          features, genome sequencing of amphibians has significantly lagged behind that of other vertebrates, primarily due to the challenges
                           of assembling their large, repeat-rich genomes and the relative lack of societal support. The emergence of long-read sequencing 
                           technologies, combined with advanced molecular and computational techniques that improve scaffolding and reduce computational 
                           workloads, is now making it possible to address some of these challenges. To promote and accelerate the production and use of 
                           amphibian genomics research through international coordination and collaboration, we launched the Amphibian Genomics Consortium
                            (AGC, https://mvs.unimelb.edu.au/amphibian-genomics-consortium) in early 2023. This burgeoning community already has more than 
                            282 members from 41 countries. The AGC aims to leverage the diverse capabilities of its members to advance genomic resources 
                            for amphibians and bridge the implementation gap between biologists, bioinformaticians, and conservation practitioners. Here 
                            we evaluate the state of the field of amphibian genomics, highlight previous studies, present challenges to overcome, and call
                             on the research and conservation communities to unite as part of the AGC to enable amphibian genomics research to “leap” to 
                             the next level.</p>
                </div>
              </div>
            </div>
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pub-8"
                >
                <p style="text-indent: -36px; padding-left: 36px;">8. Gustafson JA et al. [including <span style="font-weight:bold">Snead AA</span>]. 
                  High-coverage nanopore sequencing of samples from the 1000 Genomes Project to build a comprehensive catalog of human genetic variation. <span class="fst-italic">Genome Research</span>
                     (<a href="https://genome.cshlp.org/content/early/2024/10/02/gr.279273.124.abstract" class="link-primary" target="_blank">doi: 10.1101/gr.279273.124</a>)</p>
                </button>
              </h2>
              <div
                id="pub-8"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Abstract:</span> Fewer than half of individuals with a suspected Mendelian or 
                      monogenic condition receive a precise molecular diagnosis after comprehensive clinical genetic testing. Improvements
                       in data quality and costs have heightened interest in using long-read sequencing (LRS) to streamline clinical genomic
                        testing, but the absence of control datasets for variant filtering and prioritization has made tertiary analysis of
                         LRS data challenging. To address this, the 1000 Genomes Project ONT Sequencing Consortium aims to generate LRS data 
                         from at least 800 of the 1000 Genomes Project samples. Our goal is to use LRS to identify a broader spectrum of variation
                          so we may improve our understanding of normal patterns of human variation. Here, we present data from analysis of the 
                          first 100 samples, representing all 5 superpopulations and 19 subpopulations. These samples, sequenced to an average depth
                           of coverage of 37x and sequence read N50 of 54 kbp, have high concordance with previous studies for identifying single 
                           nucleotide and indel variants outside of homopolymer regions. Using multiple structural variant (SV) callers, we identify
                            an average of 24,543 high-confidence SVs per genome, including shared and private SVs likely to disrupt gene function as
                             well as pathogenic expansions within disease-associated repeats that were not detected using short reads. Evaluation of
                              methylation signatures revealed expected patterns at known imprinted loci, samples with skewed X-inactivation patterns,
                               and novel differentially methylated regions. All raw sequencing data, processed data, and summary statistics are 
                               publicly available, providing a valuable resource for the clinical genetics community to discover pathogenic SVs.</p>
                </div>
              </div>
            </div>
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pub-7"
                >
                <p style="text-indent: -36px; padding-left: 36px;">7. <span style="font-weight:bold">Snead AA</span>,
                    Tatarenkov A, Taylor DS, Marson K, Earley RL (2024). Centrality to the metapopulation is more important for population genetic diversity than habitat area or fragmentation. <span class="fst-italic">Biology Letters</span> 20 
                     (<a href="https://royalsocietypublishing.org/doi/abs/10.1098/rsbl.2024.0158" class="link-primary" target="_blank">doi: 10.1098/rsbl.2024.0158</a>) 
                     (<a href="https://github.com/anthonysnead/Rivulus-Genetic-Diversity" class="link-primary" target="_blank">Code</a>)</p>
                </button>
              </h2>
              <div
                id="pub-7"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Abstract:</span> Drift and gene flow affect genetic diversity. Given that the strength of
                       genetic drift increases as population size decreases, management activities have focused on increasing population size 
                       through preserving habitats to preserve genetic diversity. Few studies have empirically evaluated the impacts of drift 
                       and gene flow on genetic diversity. (<span class="fst-italic">Kryptolebias marmoratus</span>), henceforth ‘rivulus’, is a small killifish restricted to 
                       fragmented New World mangrove forests with gene flow primarily associated with ocean currents. Rivulus form distinct 
                       populations across patches, making them a well-suited system to test the extent to which habitat area, fragmentation and
                        connectivity are associated with genetic diversity. Using over 1000 individuals genotyped at 32 microsatellite loci, 
                        high-resolution landcover data and oceanographic simulations with graph theory, we demonstrate that centrality 
                        (connectivity) to the metapopulation is more strongly associated with genetic diversity than habitat area or 
                        fragmentation. By comparing models with and without centrality standardized by the source population’s genetic diversity,
                         our results suggest that metapopulation centrality is critical to genetic diversity regardless of the diversity of 
                         adjacent populations. While we find evidence that habitat area and fragmentation are related to genetic diversity, 
                         centrality is always a significant predictor with a larger effect than any measure of habitat configuration.                  dispersal via wind or ocean currents can drive asymmetric gene flow, which influences patterns of genetic
                   variation and the capacity of populations to evolve in response to environmental change.</p>
                </div>
              </div>
            </div>
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pub-6"
                >
                <p style="text-indent: -36px; padding-left: 36px;">6. <span style="font-weight:bold">Snead AA</span>,
                    Tatarenkov A, Avise JC, Taylor DS, Turner BJ, Marson K, Earley RL (2023). Out to sea: Ocean currents and patterns
                     of asymmetric gene flow in an intertidal fish species. <span class="fst-italic">Frontiers in Genetics</span> 14 
                     (<a href="https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2023.1206543/full" class="link-primary" target="_blank">doi: 10.3389/fgene.2023.1206543</a>) 
                     (<a href="https://github.com/anthonysnead/Rivulus_Ocean_Current_Gene_Flow_2023" class="link-primary" target="_blank">Code</a>)</p>
                </button>
              </h2>
              <div
                id="pub-6"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Abstract:</span> Passive dispersal via wind or ocean currents can drive asymmetric gene flow, which influences patterns of genetic
                   variation and the capacity of populations to evolve in response to environmental change. The mangrove rivulus fish 
                   (<span class="fst-italic">Kryptolebias marmoratus</span>), hereafter “rivulus,” is an intertidal fish species restricted to the highly fragmented New 
                   World mangrove forests of Central America, the Caribbean, the Bahamas, and Florida. Mangrove patches are biological 
                   islands with dramatic differences in both abiotic and biotic conditions compared to adjacent habitat. Over 1,000 individual
                    rivulus across 17 populations throughout its range were genotyped at 32 highly polymorphic microsatellites. Range-wide 
                    population genetic structure was evaluated with five complementary approaches that found eight distinct population clusters.
                     However, an analysis of molecular variance indicated significant population genetic structure among regions, populations 
                     within regions, sampling locations within populations, and individuals within sampling locations, indicating that rivulus 
                     has both broad- and fine-scale genetic differentiation. Integrating range-wide genetic data with biophysical modeling 
                     based on 10 years of ocean current data showed that ocean currents and the distance between populations over water drive 
                     gene flow patterns on broad scales. Directional migration estimates suggested some significant asymmetries in gene flow 
                     that also were mediated by ocean currents and distance. Specifically, populations in the center of the range (Florida Keys)
                      were identified as sinks that received migrants (and alleles) from other populations but failed to export individuals. 
                      These populations thus harbor genetic variation, perhaps even from extirpated populations across the range, but ocean 
                      currents and complex arrangements of landmasses might prevent the distribution of that genetic variation elsewhere. 
                      Hence, the inherent asymmetry of ocean currents shown to impact both genetic differentiation and directional migration 
                      rates may be responsible for the complex distribution of genetic variation across the range and observed patterns of 
                      metapopulation structure.</p>
                </div>
              </div>
            </div>
            <!-- Item 2 -->
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pub-5"
                >
                <p style="text-indent: -36px; padding-left: 36px;">5. Clark RD, Fitz KS, Garcia E, Jaynes KE, Reid BN, Sawkins A, 
                    <span style="font-weight:bold">Snead AA</span>,
                    Whalen J, Pinsky ML (2023) The Practice and Promise of Temporal Genomics for Measuring Evolutionary 
                    Responses to Global Change. <span class="fst-italic">Molecular Ecology Resources</span> 
                     (<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/1755-0998.13789" class="link-primary" target="_blank">doi: 10.1111/1755-0998.13789</a>) 
                     (<a href="https://github.com/TempGenomics-RCN/StateofField_Analyses" class="link-primary" target="_blank">Code</a>)</p>
                </button>
              </h2>
              <div
                id="pub-5"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Abstract:</span> Understanding the evolutionary consequences of anthropogenic change 
                        is imperative for estimating long-term species resilience. While contemporary genomic data can provide us with 
                        important insights into recent demographic histories, investigating past change using present genomic data alone 
                        has limitations. In comparison, temporal genomics studies, defined herein as those that incorporate time series 
                        genomic data, utilize museum collections and repeated field sampling to directly examine evolutionary change. As 
                        temporal genomics is applied to more systems, species and questions, best practices can be helpful guides to make 
                        the most efficient use of limited resources. Here, we conduct a systematic literature review to synthesize the 
                        effects of temporal genomics methodology on our ability to detect evolutionary changes. We focus on studies 
                        investigating recent change within the past 200 years, highlighting evolutionary processes that have occurred 
                        during the past two centuries of accelerated anthropogenic pressure. We first identify the most frequently studied 
                        taxa, systems, questions and drivers, before highlighting overlooked areas where further temporal genomic studies may 
                        be particularly enlightening. Then, we provide guidelines for future study and sample designs while identifying key
                         considerations that may influence statistical and analytical power. Our aim is to provide recommendations to a broad
                          array of researchers interested in using temporal genomics in their work.</p>
                </div>
              </div>
            </div>
            <!-- Item 3 -->
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pub-4"
                >
                <p style="text-indent: -36px; padding-left: 36px;">4. <span style="font-weight:bold">Snead AA</span> & Clark RD (2022). The Biological Hierarchy, Time, and Temporal ’Omics in Evolutionary Biology:
                     A Perspective. <span class="fst-italic">Integrative & Comparative Biology</span> 62(6), 1872-1886 (<a href="https://academic.oup.com/icb/article/62/6/1872/6691691" class="link-primary"
                      target="_blank">doi: 10.1093/icb/icac138</a>)</p>
                </button>
              </h2>
              <div
                id="pub-4"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Abstract:</span> Sequencing data—genomics, transcriptomics, epigenomics, proteomics,
                         and metabolomics—have revolutionized biological research, enabling a more detailed study of processes, ranging from 
                         subcellular to evolutionary, that drive biological organization. These processes, collectively, are responsible for
                          generating patterns of phenotypic variation and can operate over dramatically different timescales (milliseconds to
                           billions of years). While researchers often study phenotypic variation at specific levels of biological organization
                            to isolate processes operating at that particular scale, the varying types of sequence data, or ‘omics, can also 
                            provide complementary inferences to link molecular and phenotypic variation to produce an integrated view of 
                            evolutionary biology, ranging from molecular pathways to speciation. We briefly describe how ‘omics has been used 
                            across biological levels and then demonstrate the utility of integrating different types of sequencing data across
                             multiple biological levels within the same study to better understand biological phenomena. However, 
                             single-time-point studies cannot evaluate the temporal dynamics of these biological processes. Therefore, we put 
                             forward temporal ‘omics as a framework that can better enable researchers to study the temporal dynamics of target
                              processes. Temporal ‘omics is not infallible, as the temporal sampling regime directly impacts inferential
                               ability. Thus, we also discuss the role the temporal sampling regime plays in deriving inferences about the
                                environmental conditions driving biological processes and provide examples that demonstrate the impact of the
                                 sampling regime on biological inference. Finally, we forecast the future of temporal ‘omics by highlighting
                                  current methodological advancements that will enable temporal ‘omics to be extended across species and 
                                  timescales. We extend this discussion to using temporal multi-omics to integrate across the biological 
                                  hierarchy to evaluate and link the temporal dynamics of processes that generate phenotypic variation.</p>
                </div>
              </div>
            </div>
            <!-- Item 4 -->
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pub-3"
                >
                <p style="text-indent: -36px; padding-left: 36px;">3. <span style="font-weight:bold">Snead AA</span> & Alda F (2022). Time Series Sequences for Evolutionary Inferences. 
                    <span class="fst-italic">Integrative & Comparative Biology</span> 62(6), 1872-1886 (<a href="https://academic.oup.com/icb/article/62/6/1771/6698718" class="link-primary"
                     target="_blank">doi: 10.1093/icb/icac146</a>)</p>
                </button>
              </h2>
              <div
                id="pub-3"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Excerpt from Introduction:</span> Natural history collections, advancing methodologies, and lower 
                        sequencing costs have propelled the field of temporal genomics. The field is diverse, with both applied 
                        (Wasko et al. 2004; Vega et al. 2017; Kotzé et al. 2019) and fundamental (Bergland et al. 2014; Machugh et al. 2017) 
                        research focused on short (Kawecki et al. 2012; Lenski et al. 2015; Durland et al. 2021) and long 
                        (Brunel et al. 2020; Der Sarkissian et al. 2020) timescales. On longer timescales, ancient DNA (aDNA) 
                        research focuses on processes that act over thousands (Machugh et al. 2017; Yang et al. 2020) to millions 
                        (van der Valk et al. 2021) of years. Loog et al. (2020) used contemporary and ancient samples of the grey wolf,
                         Canis lupus, spanning 50,000 years to trace modern wolf ancestry back to an expansion from Beringia 
                         (the land bridge spanning Asia and North America). On a shorter timescale, Perry et al. (2022), this issue, 
                         quantify gene expression in brown bears, Ursus arctos, in different tissue types to investigate differences 
                         in gene isoform expression among seasons of the hibernation cycle. By sampling repeatedly within a year, the 
                         authors uncovered specific changes in gene expression that relate to hibernation phenotypes, providing insights 
                         into the molecular mechanisms that regulate complex animal behaviors. Furthermore, temporal methodologies can use
                          samples across development to better understand changes in gene expression throughout an organism's life. In this 
                          issue, Oomen et al. (2022) used experimental tanks of Atlantic cod (Gadus morhua) larvae at different temperatures 
                          and sampled them throughout development to examine the temporal dynamics of gene expression under thermal stress. 
                          Through this special issue, we aim to highlight the diversity of temporal genomic research while promoting the
                           extension of time-series methodologies to other “omics” approaches (e.g., metabolomics, proteomics, epigenomics).
                            This introduction provides a brief overview of the application of temporal genomics to study microevolution, and
                             its implementation within conservation. We also contextualize the studies within this special issue to inspire 
                             future research with time-series molecular data.</p>
                </div>
              </div>
            </div>
            <!-- Item 5 -->
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#pub-2"
                >
                <p style="text-indent: -36px; padding-left: 36px;">2. <span style="font-weight:bold">Snead AA</span> & Earley RL (2022). 
                    Predicting the in-Between: Present and future habitat suitability of an intertidal euryhaline fish. 
                    <span class="fst-italic">Ecological Informatics</span> 68:101523 (<a href="https://www.sciencedirect.com/science/article/pii/S1574954121003149" class="link-primary"
                    target="_blank">doi: 10.1016/j.ecoinf.2021.101523</a>). (<a href="https://github.com/anthonysnead/Rivulus-Future-Suitability-Niche-Model" class="link-primary" target="_blank">Code</a>)
                </button>
              </h2>
              <div
                id="pub-2"
                class="accordion-collapse collapse"
                data-bs-parent="#publications"
              >
                <div class="accordion-body">
                  <span style="font-weight:bold">Abstract:</span> New World mangrove trees are foundation species, and their range is predicted to expand northward with 
                    climate change. Foundation species are commonly prioritized for conservation, with the goal of preserving the entire 
                    community that depends on them. However, no studies have explicitly investigated whether mangrove-dependent species'
                     ranges will track the northward expansion of New World mangrove forests. We use the mangrove rivulus fish, 
                     <span class="fst-italic">Kryptolebias marmoratus</span>, to investigate shifts in habitat suitability in response to various climate change 
                     scenarios (Representative Concentration Pathways 2.6, 4.5, 6.0, and 8.5). Niche models for coastal species focus on 
                     traditional climatic variables (e.g., precipitation, temperature) even though coastal habitats also are directly
                      influenced by marine variables (e.g., sea surface salinity). We employ a novel data integration method that combines
                       marine and climatic variables, and that accounts for model selection uncertainty using model averaging to provide
                        robust estimates of habitat suitability. Contrary to expectation, suitability of rivulus habitat is predicted to
                         increase in the south and decrease or remain unchanged in the north across all climate change scenarios. Thus, 
                         rivulus might experience range contraction, not expansion. Habitat became more suitable with increased salinity
                          of the saltiest month and precipitation of the driest quarter. In laboratory settings, rivulus have higher survival,
                           reproductive success, and growth rates in low salinities. This discrepancy suggests that some combination of the 
                           responses of rivulus and its competitors to environmental change will restrict rivulus to habitats that laboratory
                            experiments consider suboptimal. Our models suggest that focusing conservation decisions on foundation species
                             could overestimate habitat availability and resilience of affiliated communities while simultaneously 
                             underestimating species declines and extinction risks.
                </div>
              </div>
            </div>
            <!-- Item 6 -->
            <div class="accordion-item">
                <h2 class="accordion-header">
                  <button
                    class="accordion-button collapsed"
                    type="button"
                    data-bs-toggle="collapse"
                    data-bs-target="#pub-1"
                  >
                  <p style="text-indent: -36px; padding-left: 36px;">1. Davidson J, Summerfelt S, Espmark AMO, Mota V, Marancik D, Earley RL, 
                    <span style="font-weight:bold">Snead AA</span>, Good C (2021). Effects of ozone on post-smolt Atlantic salmon (<span class="fst-italic">Salmo salar</span>) 
                    performance, health, and maturation in freshwater recirculation aquaculture systems. <span class="fst-italic">Aquaculture</span> 533:736208
                     (<a href="https://www.sciencedirect.com/science/article/pii/S0044848620339144" class="link-primary"
                     target="_blank">doi: 10.1016/j.aquaculture.2020.736208</a>).</p>
                  </button>
                </h2>
                <div
                  id="pub-1"
                  class="accordion-collapse collapse"
                  data-bs-parent="#publications"
                >
                  <div class="accordion-body">
                      <p><span style="font-weight:bold">Abstract:</span> Steroid hormones accumulate in recirculation aquaculture 
                        systems (RAS) and may influence the reproductive physiology of farmed fish. Ozone reduces hormone concentrations
                         in freshwater RAS used to rear Atlantic salmon, but its effect on reproductive development is unknown.
                          Accordingly, an 8-month trial was carried out to evaluate the growth, health, and maturation of post-smolt
                           Atlantic salmon (296 ± 4 g initial weight) reared in six replicated freshwater RAS (9.5 m3 total volume) 
                           operated with or without ozone (N = 3/treatment). Residual ozone was controlled with an oxidation reduction
                            potential (ORP) of 300–320 mV, and mean water temperature was maintained at 14.7 °C. Atlantic salmon growth
                             was generally faster in ozonated RAS. Salmon from RAS with and without ozone weighed 2156 ± 101 and 1810 ± 
                             15 g, respectively, by the end of the study. Caudal, anal, and pelvic fin damage was greater (P < 0.05) for
                              salmon in ozonated RAS early in the trial but improved thereafter. No statistical differences in gill, skin,
                               and skeletal muscle histopathology were observed between treatments at the end of the study. Waterborne 
                               estradiol, testosterone, and 11-ketotestosterone levels were periodically lower (P < 0.05) in ozonated RAS,
                                but maturing salmon were more prevalent in these systems. At the end of the trial, percent maturation of
                                 salmon populations reared in RAS with and without ozone was 63 ± 7 and 48 ± 1%, respectively; however,
                                  maturity appeared to be related to fish size. Improved water quality was observed in ozonated RAS including
                                   reduced dissolved copper, iron, and zinc levels, total heterotrophic bacteria counts, and true color, and
                                    increased ultraviolet transmittance, which may have supported improved Atlantic salmon growth. Overall, 
                                    ozone did not inhibit the onset or prevalence of Atlantic salmon maturation, but significant improvements
                                     in water quality and salmon growth performance resulted from its use.</p>
                  </div>
                </div>
              </div>
          </div>
        </div>
      </section>

           <!-- Special Issues -->
           <section id="Special-Issues" class="pt-2">
            <div class="container">
              <h2 class="text-center mb-4"><u>Special Issues</u></h2>
              <div class="accordion custom-accordion accordion-flush" id="Special-Issues">
                <!-- Item 1 -->
                <div class="accordion-item">
                  <h2 class="accordion-header">
                    <button
                      class="accordion-button collapsed"
                      type="button"
                      data-bs-toggle="collapse"
                      data-bs-target="#SI-1"
                    >
                    <p style="text-indent: -36px; padding-left: 36px;">1. Organizer/Associate Editor, <span class="fst-italic">Integrative & Comparative Biology</span>, 
                        2022. Special issue on “Temporal Genomics.”</p>
                    </button>
                  </h2>
                  <div
                    id="SI-1"
                    class="accordion-collapse collapse"
                    data-bs-parent="#Special-Issues"
                  >
                    <div class="accordion-body">
                  </div>
                </div>
              </div>
        </div>
          </section>

     <!-- Media Coverage -->
     <section id="Media-Coverage" class="pt-2">
        <div class="container">
          <h2 class="text-center mb-4"><u>Media Coverage</u></h2>
          <div class="accordion accordion-flush" id="Media-Coverage">
            <!-- Item 1 -->
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#Media-1"
                >
                <p style="text-indent: -36px; padding-left: 36px;">1. Schmelzer H. 2022. Microevolution can guide conservation efforts. 
                    <a href="https://issuu.com/naturevolve" class="link-primary" target="_blank">NatureVolve</a> 11:26-29 <br>
                    <span class="fst-italic">Interview & Popular Science Article Spotlight</span>
                </p>
                </button>
              </h2>
              <div
                id="Media-1"
                class="accordion-collapse collapse"
                data-bs-parent="#Media-Coverage"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Excerpt:</span> The rivulus fish is native to mangrove forests in Florida, the Caribbean, and Central
                        America. Most individuals in this species are hermaphrodites, allowing them to
                        reproduce on their own. Because of this quality, and their tolerance to a wide
                        range of environmental conditions, rivulus is an important species in researching
                        genetics, ecology, and evolution. One question that studying rivulus helps answer
                        is how species adjust in harshly changing environmental conditions. Mangrove
                        forests in which this fish lives are located in saltwater or brackish water in coastal
                        intertidal zones. The forests experience changing tide levels as well as have general
                        susceptibility to changing environmental conditions such as salinity, temperature,
                        and precipitation.</p>
                </div>
            </div>
          </div>
        </div>
    </div>
      </section>

           <!-- General Publications -->
     <section id="General-Pubs" class="pt-2">
        <div class="container">
          <h2 class="text-center mb-4"><u>General Publications</u></h2>
          <div class="accordion accordion-flush" id="General-Pubs">
            <!-- Item 1 -->
            <div class="accordion-item">
              <h2 class="accordion-header">
                <button
                  class="accordion-button collapsed"
                  type="button"
                  data-bs-toggle="collapse"
                  data-bs-target="#Gen-1"
                >
                <p style="text-indent: -36px; padding-left: 36px;"><span style="font-weight:bold">1. Snead AA</span> 2023. Networking and Establishing Successful Collaborations. 
                    Early Ecologist Section. <a href="https://www.esa.org/earlycareer/2023/06/16/networking-and-establishing-successful-collaborations/" class="link-primary" target="_blank">
                      https://www.esa.org/earlycareer/2023/06/16/networking-and-establishing-successful-collaborations/</a>.
                </p>
                </button>
              </h2>
              <div
                id="Gen-1"
                class="accordion-collapse collapse"
                data-bs-parent="#General-Pubs"
              >
                <div class="accordion-body">
                    <p> <span style="font-weight:bold">Excerpt:</span> “Network! Network! Network!”
                         I can hear the phrase ringing in my ears, and it transports me back to when I was a 
                         fresh PhD student in my advisor’s office. As early career scientists, we have been told 
                         that networking is important, and I took that to heart. I jumped at every chance to meet 
                         new people, even when I was already stretched too thin, but why? There are nontangible reasons
                          like developing interpersonal skills and making good impressions to hopefully aid in future career
                           transitions. However, there is also a more immediate reason, collaborations. We all know that 
                           science is collaborative, and strong collaborations can make or break your research. However, 
                           no one ever tells you how to transition your network into meaningful collaborations. While not 
                           all relationships are easily expanded into collaborative projects, I have some tips and suggestions,
                            that I learned the hard way, to grow your relationships into collaborations that, hopefully, you 
                            maintain throughout your career.</p>
                </div>
              </div>
            </div>
          </div>
          </div>
    </div>
</div>
      </section>
      

    <!-- White Space -->
<section id="white-space" class="p-5 bg-light text-light">
  <div class="container">
    </div>
  </div>
</section>

    <!-- Footer -->
    <footer class="p-5 bg-dark text-white text-center position-relative">
      <div class="container">
        <p class="lead">Made with Bootstrap - <a href="https://github.com/anthonysnead/anthonysnead.github.io" 
          class="link-primary" target="_blank">
          See the Code!</a></p>
        <a href="#" class="position-absolute bottom-0 end-0 p-5">
          <i class="bi bi-arrow-up-circle h1"></i>
        </a>
      </div>
    </footer>
    </body>
    </html>


